.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,18 @@
-## Contributor Comments
-[Please place any comments here.  A description of the problem/enhancement, how to reproduce the issue, your testing methodology, etc.]
+*Thank you for submitting a contribution to the zeek-kafka writer plugin.
 
+In order to streamline the review process, we ask you follow [these guidelines](https://github.com/SeisoLLC/zeek-kafka/blob/main/.github/CONTRIBUTING.md) and complete the following template.*
 
-## Pull Request Checklist
+# Summary of the contribution
+...
 
```

Comment:
Versions tested with:  zeek / docker

Reply:
I have a follow on that runs this in CI so I don't feel the need to duplicate manually, what do you think?  Draft PR @ #9